paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/DefaultEventReporter.kt

```diff
@@ -480,4 +503,9 @@ internal class DefaultEventReporter @Inject internal constructor(
             null
         }
     }
+
+    private companion object {
+        const val CLIENT_ID = "stripe-mobile-sdk"
+        const val ORIGIN = "stripe-mobile-sdk-android"
+    }
```

Comment:
We're reusing this client id originally made for MPE - https://stripe.sourcegraphcloud.com/stripe-internal/zoolander/-/blob/src/resources/com/stripe/dscore/analyticseventlogger/server/rpcserver/client_config.yaml?L1667-1674